acquisitions/src/main/resources/thunderjet/mod-orders/features/cancel-and-delete-order.feature

```diff
@@ -7,27 +7,27 @@ Feature: Cancel and delete order
 
     * url baseUrl
 
-    * callonce login testAdmin
+    * call login testAdmin
     * def okapitokenAdmin = okapitoken
-    * callonce login testUser
+    * call login testUser
     * def okapitokenUser = okapitoken
```

Comment:
> Do you change it because there is only one scenario?

Yes
> Is it expected then for one Feature there should be only Scenario?

No, a feature can have many scenarios. The many scenarios present prior to my changes didn't convey the intent of the test.
> And in case if it's possible to have several scenarios in one feature then I think it make sense to use callonce for stuff that should be executed only once. Am I right?

Yes, it makes sense in that scenario to use `callonce` when several scenarios are involved.